nss-day-cohort-48/rare-client-river-rats
# updated Rare User Details

#### Changes Made


1. Modified file modified:   src/components/ApplicationViews.js
       src/components/RareUser/RareUserDetail.js
      src/components/RareUser/RareUserList.js
       src/components/RareUser/RareUserSearch.js and functionality.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout NWM fist client
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Instructions : started the rare users form and also the search for users, still some code that needs to be updated. this will be continually worked on .
   > working on getting additional user details to show
4. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
